src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -48,7 +48,7 @@ public class BuildParameters : ITranslatable
         /// <summary>
         /// The shutdown timeout for the request builder.
```

Comment:
```suggestion
        /// The shutdown timeout for the request builder in milliseconds.
```
I'd suggest applying this to each variable that stores an "amount of time", or appending `// 90 seconds` as other similar vars have done.